sdk/eventhubs/azure-eventhubs/src/test/java/com/azure/messaging/eventhubs/SendOptionsTest.java

```diff
@@ -27,9 +28,9 @@ public void setMaximumSize() {
         int size = 1024;
         SendOptions options = new SendOptions();
 
-        options.maximumSizeInBytes(size);
+        // options.maximumSizeInBytes(size);
 
-        Assert.assertEquals(size, options.maximumSizeInBytes());
+        // Assert.assertEquals(size, options.maximumSizeInBytes());
```

Comment:
Should these be commented out? If yes, it would be better to just delete them if they are no longer required.

Reply:
yes, so @conniey is going to be making future updates related to this and these would be replaced anyway so wasn't much point fixing them now.

Reply:
Yes. I told @samvaity to comment out these tests. My PR #4403 removes these properties and fixes these tests.